### PR TITLE
Spelling and grammar correcions were made to comments contained withi…

### PR DESCRIPTION
**Pull request by purifyyourhome**
…n files in the CDLINK directory.

### CODE REVIEW

File: SOURCES/src/cdlink/CDLINK.ASM [hunk 1]
```diff
@@ -909,7 +909,7 @@ f11bad:
 	jmp	i21fcberr
 
 getfcbnm proc	near
-	lea	si,[bx+1]		; save fcb filename to sWSName1
+	lea	si,[bx+1]		; save fcb file name to sesame1
```
Comment (andrewbird) [inline SOURCES/src/cdlink/CDLINK.ASM]: This doesn't look right, I guess you used a spell checker?

File: SOURCES/src/cdlink/CDLINK.ASM [hunk 2]
```diff
@@ -13,14 +13,14 @@
 ;	    Edit out reporting of "." and ".." in a root directory.  ;
 ;	    Adjust for "." in a path name.			     ;
 ;   9/01/92 Do stack switch in Executor.  Correct "copyname" proc    ;
-;	    to truncate target filename.ext to 8.3 characters.	     ;
+;	    to truncate target file name.ext to 8.3 characters.	     ;
```
Comment (andrewbird) [inline SOURCES/src/cdlink/CDLINK.ASM]: In this instance the original was correct as `filename.ext` is intended to be in '8 characters' + '.' + 3 characters format.

**Comment by roelandjansen** [comment]
@andrewbird -- I'm sort of busy (with CKA coourse) -- could you review it?

**Comment by roelandjansen** [comment]
and @purifyyourhome  thanks for the add's, it's  appreciated.

**Comment by andrewbird** [comment]
@roelandjansen it looks fine. All changes are commentary except one that's inside a display string. So whilst I haven't compiled it, I believe it should do so.

**Comment by roelandjansen** [comment]
@andrewbird  ok, I'll merge it. 

Can you check if you have approval rights before I do?

**Comment by andrewbird** [comment]
I have only normal user rights, but I'm happy with that.

**Pull request merged**